region/norfair/east/Cathedral.json

```diff
@@ -727,7 +727,8 @@
         }},
         "h_heatProof",
         {"refill": ["Energy", "Missile", "Super"]}
-      ]
+      ],
+      "resetsObstacles": ["A"]
```

Comment:
A is the geruta, with heatproof it could probably be cleared. not sure which is better

Reply:
Yeah for this farm I think I had in mind just farming the one Sova near the door, since the Geruta can be a pain to deal with if you don't have a good beam. So probably leave the Geruta reset I guess? I notice there's not even any strat in logic to kill it with a weak beam, so that should probably be looked into at some point.